README.md

```diff
@@ -172,6 +174,38 @@ The `pm-x` generator generates commands for running a package through JavaScript
 - `name`: The package name (by default tries to read from the `name` field in `package.json`).
 - `usage`: An additional string appended at the end of each command suggesting usage. (defaults to `""`).
 
+### `badges`
+
+The `badges` generator generates badges for npm version, npm downloads and some optional ones like codecov and bundle.
+
+#### Usage
+
+    <!-- automd:badges name=package-name -->
+    <!-- /automd -->
+
+**Updated Result:**
+
+    <!-- automd:badges name=package-name -->
+
+    [![npm version](https://img.shields.io/npm/v/package-name?style=flat&colorA=18181B&colorB=F0DB4F)](https://npmjs.com/package/package-name)
+    [![npm downloads](https://img.shields.io/npm/dm/package-name?style=flat&colorA=18181B&colorB=F0DB4F)](https://npmjs.com/package/package-name)
+
+    <!-- /automd -->
+
+#### Arguments
+
+- `name`: The package name (by default tries to read from the `name` field in `package.json`).
+- `style`: The badge style (defaults to `flat`).
+- `colorA`: The left color of the badge (defaults to `18181B`).
+- `colorB`: The right color of the badge (defaults to `F0DB4F`).
```

Comment:
Something about `colorA` and `colorB` doesn't sit right with me. What about `color-left` and `color-right`? Is that too complicated? I think it also makes it more self-explanatory.

EDIT: Huh reading through I guess those are the names of the actual image's query params...

EDIT2: Those query params are not documented though, maybe deprecated in favor of the `labelColor` and `color` params? Maybe `label-color` and `bg-color` could be the argument names?

Reply:
Should be good now 👍🏼